src/Files.App/Helpers/FilePropertiesHelpers.cs

```diff
@@ -118,5 +121,21 @@ void Navigate(Frame frame)
 				frame.Navigate(typeof(Properties), argument, new SuppressNavigationTransitionInfo());
 			}
 		}
+
+		private static string GetFilesLogoPath()
+		{
+			var appTilesPath = Path.Combine(Package.Current.InstalledLocation.Path, "Assets/AppTiles");
+
+			if (Directory.Exists(Path.Combine(appTilesPath, "Dev")))
+				return Path.Combine(appTilesPath, "Dev", "Logo.ico");
```

Comment:
Since the Combine for `appTilePath` and `Dev` is made two times one after another, maybe it could be worth to store it inside a variable? I am not sure if it's useful or not, just to eliminate some duplication.

```suggestion
                        var appTilesDevPath = Path.Combine(appTilesPath, "Dev");
			if (Directory.Exists(appTilesDevPath))
				return Path.Combine(appTilesDevPath, "Logo.ico");
```

Reply:
I modified GetFilesLogoPath with Linq and a sub function.